### PR TITLE
main bump hdk 0.3.1 hdi 0.4.1

### PR DESCRIPTION
**Pull request by dauphin3**

### CODE REVIEW

File: flake.lock
Comment (mattyg) [inline flake.lock:1]: we should keep flake.lock in the repo

File: package.json [hunk 1]
```diff
@@ -50,5 +51,8 @@
   },
   "engines": {
     "npm": ">=7.0.0"
+  },
+  "dependencies": {
+    "vue-tsc": "^2.0.29"
```
Comment (mattyg) [inline package.json:56]: looks like the error you're getting in tryorama in related to vue-tsc version? I think this should only be a dependency in the `ui/package.json` -- so we don't need it here (also looks like they are on conflicting versions).

**Review by mattyg — COMMENTED** [review]
I think you may also need to bump the tryorama version for compatability with holochain 0.3.x

**Comment by jost-s** [comment]
I don't understand why clippy is failing on macos. I can't reproduce that.

**Comment by mattyg** [comment]
> I don't understand why clippy is failing on macos. I can't reproduce that.

I made a PR in holochain-wasmer to hopefully resolve it (that's where test-fuzz-internal is coming from): https://github.com/holochain/holochain-wasmer/pull/114

I think we can merge it for now since its def an upstream issue.

**Comment by jost-s** [comment]
Cool. Funny though that I don't have that error locally, isn't it?

**Pull request merged**